conda/compat.py

```diff
@@ -60,11 +110,9 @@ def lchmod(path, mode):
 
     # Modified from http://hg.python.org/cpython/file/3.3/Lib/tempfile.py. Don't
     # use the 3.4 one. It uses the new weakref.finalize feature.
-    import shutil as _shutil
-    import warnings as _warnings
-    import os as _os
     from tempfile import mkdtemp
     range = xrange
+    # used elsewhere - do not remove
     from itertools import izip as zip
 
     class TemporaryDirectory(object):
```

Comment:
Shouldn't we remove this whole class here since it looks like an exact duplicate of the one above?